scaling_performance/scaling_cluster_monitoring_operator.adoc

```diff
@@ -0,0 +1,152 @@
+[[scaling-performance-cluster-metrics]]
+= Scaling Cluster Metrics
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
```

Comment:
by the link:https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator]